administering_a_cluster/dedicated-admin-role.adoc

```diff
@@ -33,18 +33,11 @@ include::modules/dedicated-managing-quotas-and-limit-ranges.adoc[leveloffset=+1]
 [id="osd-installing-operators-from-operatorhub_{context}"]
 == Installing Operators from the OperatorHub
 
-{product-title} administrators can install Operators from a curated list
-provided by the OperatorHub. This makes the Operator available to all developers
-on your cluster to create Custom Resources and applications using that Operator.
+{product-title} administrators can install Operators from a curated list provided by the OperatorHub. This makes the Operator available to all developers on your cluster to create Custom Resources and applications using that Operator.
```

Comment:
is `a curated list provided by` redundant? In other words, should we just say you can install from OperatorHub?

Reply:
@billmvt That does sound better. Thanks for the review!